src/Controllers/DriftController.php

```
@@ -239,4 +239,68 @@ public function reviewChange($id): void
+     */
+    public function alignmentHistory(): void
+    {
+        $user      = $this->auth->user();
+        $orgId     = (int) $user['org_id'];
+        $projectId = (int) $this->request->get('project_id', 0);
+        $project = ProjectPolicy::findViewableProject($this->db, $user, $projectId);
+        if ($project === null) {
+            $this->response->redirect('/app/home');
+            return;
+        }
+        $allAlerts = DriftAlert::findByProjectId($this->db, $projectId);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\StratFlow\Models\DriftAlert' in method 'alignmentHistory'. (undefined)

(StaticAccess)